From c6293f935e1f6baf373a557a3810c1c78aa1301f Mon Sep 17 00:00:00 2001
From: xiaoyao <xiaoyao@rock-chips.com>
Date: Tue, 2 Aug 2016 18:13:02 +0800
Subject: [PATCH] arm64: dts: rockchip: modify sd minimum frequency for
 rk3399-box

The host can issue continuous clock in the frequency range
of 100KHz-400KHz for card identification mode.

Change-Id: I4e6a96a5ec49ee102a3b24c3f7f9b74d6bc8fd5f
Signed-off-by: xiaoyao <xiaoyao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index fcfd274def13..97b6df177490 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -208,7 +208,7 @@
 
 &sdmmc {
 	clock-frequency = <100000000>;
-	clock-freq-min-max = <400000 100000000>;
+	clock-freq-min-max = <100000 100000000>;
 	supports-sd;
 	bus-width = <4>;
 	cap-mmc-highspeed;
-- 
2.35.3

